pkg/trace/features.go

```diff
@@ -18,6 +18,7 @@ const (
 	DM_SUBMISSION  Feature = "dm.submission" // dimensional metrics submission
 	METRIC_MATCHER Feature = "metric.match"  // match metric by rule
 	INVENTORY      Feature = "inventory"
+	LOG_FWD        Feature = "log.fw"
```

Comment:
this is what needs to be written by the user, ie:
```
trace:
  - log.fw
```

maybe we should be more verbose on the name? like `log.forwarder`?